installation/synology.md

```diff
@@ -3,10 +3,113 @@ layout: documentation
 title: Synology DiskStation
 ---
 
-# Synology DiskStation
+# Synology DiskStation DSM 7.0 and higher
 
-The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages.
-We are proud to be able to provide an [openHAB Synology package](https://github.com/openhab/openhab-syno-spk).
+The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages. For versions of DSM before 7.0 please refer to the second half of this page.
```

Comment:
Each sentence in Markdown should be placed on a new line so it easier to review and compare changes.

```suggestion
The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages.
For versions of DSM before 7.0 please refer to the second half of this page.
```

Reply:
Tnx for the review, very useful feedback.